### Hypothesis Test

In [18]:
import numpy as np
import pandas as pd
import scipy.stats as stats

Ho = Null Hypothesis = mean is equal to 120mm Hg (Ho: mu = 120) <br>
Ha = Alternative Hypothesis = mean is not equal to 120mm Hg (Ha: mu != 120)

In [24]:
n = 100
sd = 21.21
mu = 120
sample_mean = 130.1

<b> Compute standard error of mean

In [25]:
sem = sd/np.sqrt(n)
sem

2.121

<b> Find z score

In [26]:
z = (sample_mean - mu)/sem
z

4.761904761904759

<b> Find critical value using a 95% confidence interval

In [29]:
zc = stats.norm.ppf(1-(0.05/2))
zc

1.959963984540054

<b> Compare z score with critical value

Since z=4.76 > zc=1.96 we reject the null and assume that the mean is not equal to 120mm Hg